build.py

```diff
@@ -270,18 +270,18 @@ def do_codecov(self) :
 		
 		profile_data_path = derived_dir + "/ProfileData/" + device_guid + "/Coverage.profdata"
 		if not os.path.isfile(profile_data_path) :
-			print ColorValues.FAIL + "Coverage data file missing! : " + profile_data_path + ColorValues.END
+			print (ColorValues.FAIL + "Coverage data file missing! : " + profile_data_path + ColorValues.END)
 			return -1
 		
 		configuration_build_dir = build_settings["CONFIGURATION_BUILD_DIR"]
 		executable_path = build_settings["EXECUTABLE_PATH"]
 		executable_file_path = configuration_build_dir + "/" + executable_path
 		if not os.path.isfile(executable_file_path) :
-			print ColorValues.FAIL + "executable file missing! : " + executable_file_path + ColorValues.END
+			print (ColorValues.FAIL + "executable file missing! : " + executable_file_path + ColorValues.END)
```

Comment:
Are the changes in this file for the python3 problem? Do you think we should make a different PR for this?